.github/workflows/winget-release.yml

```diff
@@ -0,0 +1,29 @@
+name: Submit Winget Package to Windows Package Manager Community Repository
+on:
+  release:
+    types: [released]
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: "Specific tag name"
+        required: false
+        type: string
+env:
+  CI: true
+jobs:
+  winget-publish:
+    name: Publish winget package
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set winget version env
+        env:
+          TAG_NAME: ${{ inputs.tag_name || github.event.release.tag_name }}
+        run: echo "WINGET_TAG_NAME=$(echo ${TAG_NAME#v})" >> $GITHUB_ENV
+      - name: Submit package to Windows Package Manager Community Repository
+        uses: vedantmgoyal2009/winget-releaser@v2
+        with:
+          identifier: F3d.F3d
+          installers-regex: '^F3D-[\d\.]+-Windows-x86_64\.exe$'
+          version: ${{ env.WINGET_TAG_NAME }}
+          release-tag: ${{ inputs.tag_name || github.event.release.tag_name }}
+          token: ${{ secrets.WINGET_ACC_TOKEN }}
```

Comment:
so we need to generate this right ?

Reply:
The github classic token with `public read repo` permission has to be generated by someone on the f3d-app org indeed, and stored as a github secret named `WINGET_ACC_TOKEN` (+ the winget-pkgs repo has to be forked by that same user)

Reply:
Why is a fork needed ?

Reply:
noted